mods/wlssb/moves.js

```diff
@@ -440,63 +440,24 @@ exports.BattleMovedex = {
 		type: "Water",
 	},
 	// Lycanium Z
-	"finishthem": {
-		accuracy: 100,
-		basePower: 0,
-		damage: 1,
-		category: "Physical",
-		desc: "OHKOs the target if user didnt take damage. %10 chance to lower all stats by 1.",
-		id: "finishthem",
-		name: "FINISH THEM",
-		pp: 5,
-		noPPBoosts: true,
-		flags: {protect: 1, mirror: 1},
-		secondary: {
-			chance: 10,
-			self: {
-				boosts: {
-					atk: -1,
-					def: -1,
-					spa: -1,
-					spd: -1,
-					spe: -1,
-				},
-			},
-		},
-		priority: -3,
-		onPrepareHit: function (target, source) {
-			this.attrLastMove('[still]');
-			this.add('-anim', source, "Extreme Evoboost", source);
-			this.add('-anim', source, "Fusion Flare", target);
-			this.add('-anim', source, "Splintered Stormshards", target);
-		},
-		onModifyMove: function (move, target) {
-			move.damage = target.maxhp;
-		},
-		beforeTurnCallback: function (pokemon) {
-			pokemon.addVolatile('finishthem');
-		},
-		beforeMoveCallback: function (pokemon) {
-			if (pokemon.volatiles['finishthem'] && pokemon.volatiles['finishthem'].lostFocus) {
-				this.add('cant', pokemon, 'FINISH THEM', 'FINISH THEM');
-				return true;
-			}
-		},
-		effect: {
-			duration: 1,
-			onStart: function (pokemon) {
-				this.add('-singleturn', pokemon, 'move: FINISH THEM');
-			},
-			onHit: function (pokemon, source, move) {
-				if (move.category !== 'Status') {
-					pokemon.volatiles['finishthem'].lostFocus = true;
-				}
-			},
+	unserious:{
+		category: "Status",
+		id: "unserious",
+		isNonstandard: true,
+		name: "Unserious",
+		pp: 30,
+		desc: "Forces the weather to hail and uses foresight and leech seed",
+		priority: 9,
+		flags: {mirror: 1, snatch: 1},
+		onHit: function (pokemon) {
+			this.clearWeather();
+			this.setWeather('hail');
+			this.useMove('leechseed', pokemon);
+			this.useMove('foresight', pokemon);
 		},
-		target: "normal",
-		type: "Rock",
-		zMovePower: 180,
-		contestType: "Tough",
+		secondary: false,
+		target: "self",
```

Comment:
Why make it Self when you are using Foresight & Leech Seed on foe

Reply:
Makes Sure the move doesn’t miss

Reply:
thats, not the appropriate way to ensure a move hits, use ``accuracy: true,`` for that

Reply:
wont this make you use the moves on yourself? not sure

Reply:
No

Reply:
I tested it and it does use it on the opponent